benchmarks/src/main/java/uk/ac/manchester/tornado/benchmarks/blackscholes/BlackScholes.java

```diff
@@ -1,218 +1,216 @@
+     * documentation for this sample. On invocation of kernel blackScholes, each
+     * work thread calculates call price and put price values for given stock
+     * price, option strike price, time to expiration date, risk free interest
+     * and volatility factor.
+     */
+    public static void main(final String[] args) {
+
+        int size = Integer.getInteger("bs.size", 16777216);
+        final int iterations = Integer.getInteger("bs.iterations", 300);
+        if (DEBUG) {
+            System.out.println("size =" + size);
+            System.out.println("iterations =" + iterations);
+        }
+
+        final BlackScholes bs = new BlackScholes(size);
+
+        final TaskSchedule tasks = new TaskSchedule("benchmark").task("blackscholes", BlackScholes::blackscholes, bs.randArray, bs.put, bs.call);
+
+        tasks.warmup();
+
+        final long start = System.nanoTime();
+        if (USE_JAVA) {
+            for (int i = 0; i < iterations; i++) {
+                if (DEBUG) {
+                    System.out.printf("iteration %d\n", i);
+                }
+                blackscholes(bs.randArray, bs.put, bs.call);
+            }
+        } else {
+            for (int i = 0; i < iterations; i++) {
+                tasks.execute();
+            }
+            tasks.syncObjects(bs.put, bs.call);
+        }
+        final long end = System.nanoTime();
+
+        final double elapsed = (end - start) * 1e-9;
+        // final double exec = elapsed - compile;
+
+        final String id = String.format("bm=blackscholes-%d-%d", size, iterations);
+
+        if (USE_JAVA) {
+            System.out.printf("%s, id=java-reference, elapsed=%.9f, per iteration=%.9f \n", id, elapsed, elapsed / iterations);
+
+        } else {
+            System.out.printf("%s, id=%s, elapsed=%.9f, per iteration=%.9f \n", id, TornadoRuntime.getProperty("benchmark.device"), elapsed, elapsed / iterations);
+            tasks.dumpProfiles();
+        }
+
+        if (DEBUG) {
+            bs.showResults(10);
+        }
+
+    }
+
+    /**
+     * @brief Abromowitz Stegun approxmimation for PHI (Cumulative Normal
+     *        Distribution Function)
+     * @param X
+     *            input value
+     */
+    final static float phi(final float X) {
+        final float c1 = 0.319381530f;
+        final float c2 = -0.356563782f;
+        final float c3 = 1.781477937f;
+        final float c4 = -1.821255978f;
+        final float c5 = 1.330274429f;
+
+        final float zero = 0.0f;
+        final float one = 1.0f;
+        final float two = 2.0f;
+        final float temp4 = 0.2316419f;
+
+        final float oneBySqrt2pi = 0.398942280f;
+
+        final float absX = Math.abs(X);
+        final float t = one / (one + (temp4 * absX));
+
+        final float y = (one - (oneBySqrt2pi * TornadoMath.exp((-X * X) / two) * t * (c1 + (t * (c2 + (t * (c3 + (t * (c4 + (t * c5))))))))));
+
+        return (X < zero) ? (one - y) : y;
+    }
+
```

Comment:
This is the only file I have changed, but git is highlighting the entire file because I am using Ubuntu. IntelliJ on ubuntu uses LF to end the file.
There are two ways to represent a line ending in a source code file, CRLF or just LF. Windows tends to default to CRLF, but the rest of the world uses just LF, so Git/Programming standards in general prefer LF.

Reply:
I still don't get why this is like a new file. We also use Ubuntu and Red-Hat based. Are you using the TornadoVM code formatter? 

We use the Eclipse code formatter plugin, and the XML is in the TornadoVM repository.  Full explanation here:

https://github.com/beehive-lab/TornadoVM/blob/master/assembly/src/docs/3_INTELLIJ.md#required-plugins

Reply:
Sure, I will check it again. I am using IntelliJ on Ubuntu.